### ISSUE TITLE
Not reproducable

### ISSUE THREAD

**Issue by soyoung2022** [OPTML-Group/Unlearn-Saliency#25]
I followed the instructions in the README file located in the classification folder, to conduct random unlearning experiments on CIFAR-10 with 10% and 50% data removal by SalUn method. However, I am struggling to reproduce the results, as my forget accuracy remains unexpectedly high. In the paper, it states that using SalUn with 10% and 50% random unlearning results in UA values between 0 and 7. However, my experiments show values exceeding 90%, which is a significant discrepancy.

For my experiments, I trained ResNet18 model for 50 epochs and saved the weights before proceeding random unlearning experiments. I generated the masks using generate_mask.py and utilized the 0.9 or 0.8 files(with_0.8.pt, with_0.9.pt), running several experiments with each to see the effects. Additionally, I set the unlearning learning rate to 0.001 and varied the unlearning epochs from 3 to 10. 

Could you please help me identify any potential misconfigurations in my setup that might be affecting my ability to replicate your results?
